### PR TITLE
Add Upollo documentation.

### PR DESCRIPTION
**Pull request by stephennancekivell**
### Proposed changes

Add the Upollo Destination documentation to go along with our new integration.

### Merge timing
- ASAP once approved?

### CODE REVIEW

File: src/connections/destinations/catalog/upollo/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Upollo Destination
+hidden: true
+private: true
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+
+[Upollo](https://upollo.ai?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} finds and converts repeat trialers account sharers and more.
```
Comment (cayden-uw) [inline src/connections/destinations/catalog/upollo/index.md:10]: Missing comma between repeat trialers and account sharers

File: src/connections/destinations/catalog/upollo/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+11% of users signing up for free trials have [already used one](https://upollo.ai/blog/turn-repeated-trials-into-growth?utm_source=segmentio&utm_medium=docs&utm_campaign=partners), and up to 45% of users [share their logins](
+https://upollo.ai/blog/grow-by-understanding-account-sharing?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) with others.
+Inviting these users to a paid account is the number 1 underutilized growth channel for SaaS businesses.
+
+This destination is maintained by Upollo. For any questions or issues with the destination, please [contact the Upollo team](https://upollo.ai/contact?utm_source=segmentio&utm_medium=docs&utm_campaign=partners).
+
+## Benefits of <destination_name> (Actions)
```
Comment (cayden-uw) [inline src/connections/destinations/catalog/upollo/index.md:18]: Should this say Upollo?

**Review by cayden-uw — APPROVED** [review]

File: src/connections/destinations/catalog/upollo/index.md (hunk 1 above)
Comment (stephennancekivell) [reply src/connections/destinations/catalog/upollo/index.md:10]: Thanks.

File: src/connections/destinations/catalog/upollo/index.md (hunk 2 above)
Comment (stephennancekivell) [reply src/connections/destinations/catalog/upollo/index.md:18]: Opps. Yes.

**Pull request merged**